<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleWidget</class>
 <widget class="QWidget" name="SampleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="sampleTopWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>331</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="widgetStyle" stdset="0">
    <string>BGStyle</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="sampleStackPage">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="rightMargin">
          <number>8</number>
         </property>
         <item row="0" column="14">
          <widget class="QPushButton" name="btnReTest">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>      复孔</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn_Mut_Hole</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="btnBatchAdd">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>批量添加</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn</string>
           </property>
          </widget>
         </item>
         <item row="0" column="16">
          <widget class="QPushButton" name="btnCancel">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>返回样本</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="12">
          <widget class="QPushButton" name="btnReQuestLisAll">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string> LIS批量获取</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn</string>
           </property>
          </widget>
         </item>
         <item row="0" column="18">
          <widget class="QPushButton" name="btnNext">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>下一步</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtnRed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="10">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="btnSaveAdd">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>      保存</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn_Save</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lblSpace">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnPre">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>上一步</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtnRed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="11">
          <widget class="QPushButton" name="btnRequestLis">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>      LIS获取</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn_request_lis</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnIncubation">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>手动孵育</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn</string>
           </property>
          </widget>
         </item>
         <item row="0" column="17">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>120</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <widget class="QPushButton" name="btnDeleteAll">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>删除所有</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn_Delete</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QPushButton" name="btnDelete">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>     删除</string>
           </property>
           <property name="btnStyle" stdset="0">
            <string>normalBtn_Delete</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblTitle">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:32px;
font-weight:bold;</string>
           </property>
           <property name="text">
            <string>t</string>
           </property>
          </widget>
         </item>
         <item row="0" column="13">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>200</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
